packages/ui-library/src/components/graph-container/index.tsx

```diff
@@ -22,6 +22,9 @@ const StyledContainerBoxWithHeight = styled.div<{ height?: number; minHeight?: n
   min-height: ${({ minHeight, height }) => minHeight || height}px;
   ${({ height }) => (height ? ` height: ${height}px;` : 'flex: 1;')}
   color: black;
+  .recharts-surface {
```

Comment:
why?

Reply:
I started testing bigger USD values for the empty graph, above 999 the labels were cut off